docs/question/index.md

```diff
@@ -39,11 +47,17 @@ questions
         `-- question.html
 ```
 
+!!! tip
+
+    Organize your questions by kind. A recommended folder structure is `topic/subtopic/question`. Folder layouts like `exam3/question12` or `Spring20/questionName` can make it harder to find and reuse questions.
```

Comment:
I'm wary of actually recommending this organization structure, mostly because I'm not actually sure how common it is. I think the other recommendation to avoid tying questions to specific terms or assessments is good though.

```suggestion
!!! tip

    Avoid using term names (e.g. `Spring20/questionName`) or assessment names (e.g. `exam3/question12`) in your question directory structure, as these can make it harder to find and reuse questions across assessments and terms.
```

Reply:
8de961fa8bacdf280e06dd8f5e73d124836d5b15